lib/logstash/outputs/rabbitmq.rb

```diff
@@ -76,20 +71,108 @@ def initialize(params)
     super
   end
 
-  # Use MarchHare on JRuby to avoid IO#select CPU spikes
-  # (see github.com/ruby-amqp/bunny/issues/95).
-  #
-  # On MRI, use Bunny.
-  #
-  # See http://rubybunny.info and http://rubymarchhare.info
-  # for the docs.
-  if RUBY_ENGINE == "jruby"
-    require "logstash/outputs/rabbitmq/march_hare"
+  public
```

Comment:
@michaelklishin noted! I agree the style here is probably very distinct with respect to what the general ruby population writes. We're working on formalizing a style guide (that will be _different_ from the 'ruby community guide' for a variety of reasons).

For now, I'm OK if we don't have explicit per-method public/private definitions. We can resolve this later once we finish the style guide.